bench/index.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+var deepEql = require('../');
+var lodashDeepEql = require('lodash.isequal');
+var assertDeepeql = require('assert').deepEql;
+var inspect = require('util').inspect;
+var Benchmark = require('benchmark');
+var benches = [];
+var mapObjRefA = {};
+var mapObjRefB = {};
+function getArguments() {
+  return arguments;
+}
+var fixtures = {
```

Comment:
Agreed, maybe just adding what has been suggested by @zubuzon would be enough for that.